### PR TITLE
add CVE-2022-25237

### PR DESCRIPTION
**Pull request by Question406**

### CODE REVIEW

File: CVE-2022-25237/ubuntu.yml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        path: /opt/Bonita
+        state: directory
+
+    - name: Copy dockerfile
+      ansible.builtin.copy:
+        src: ./data/docker-compose.yml
+        dest: /opt/Bonita
+
+    - name: Start Docker Compose services
+      command: docker compose up -d --build
```
Comment (olegbck) [inline CVE-2022-25237/ubuntu.yml:45]: Please use community.docker.docker_compose module instead of shell commands

File: CVE-2022-25237/cvex.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+blueprint: ubuntu2204
```
Comment (olegbck) [inline CVE-2022-25237/cvex.yml:1]: Use the ubuntu2204-ubuntu2204 blueprint. Install Bonita on ubuntu1, copy the exploit to ubuntu2, run the exploit on ubuntu2 that would attack ubuntu1. Make sure that router_raw.pcap captures the network traffic. The goal is to get the traffic!

File: CVE-2022-25237/data/docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+version: '3.8'
+
+services:
+  bonita:
+    image: bonita:7.13.0
```
Comment (olegbck) [inline CVE-2022-25237/data/docker-compose.yml:5]: Please download the Docker image and keep it in the data folder. Install it from the file.

File: CVE-2022-25237/cvex.yml [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+ubuntu:
+  playbook: ubuntu.yml
+  trace: "python3"
+  command:
+    - "cd /opt/exploit/exploitcode && ls" 
+    - "python3 exploit.py test test http://localhost:8080/bonita \"cat /etc/passwd\""
```
Comment (olegbck) [inline CVE-2022-25237/cvex.yml:7]: It fails when I try it:

> 2024-11-19 08:35:52,336 - CRITICAL - [main] Command failed: <UnexpectedExit: cmd='strace -o /tmp/cvex/ubuntu_strace_python3_1.log python3 exploit.py test test http://localhost:8080/bonita "cat /etc/passwd"' exited=2>

Have you tested it?

**Comment by Question406** [comment]
Close this request due to incorrect implementation

**Pull request closed without merging**